### ISSUE TITLE
Multiple vector import does not work

### ISSUE THREAD

**Issue by madi** [OSGeo/grass#1885]
From the GRASS 8 GUI, 

File - Import vector data - Simplified import with reprojection [v.import]

and 

File - Import vector data - Import of common vector formats [v.in.ogr]

For both, the multiple import does not work. It is only possible to import one file at a time, if you select Source type:directory the list of the layers is not loaded.

**Comment by veroandreo** [comment]
I tried and it works just fine for me with a folder full of shp files... Does it happen with all formats?

![image](https://user-images.githubusercontent.com/20075188/134363330-cfe1e2f8-93d4-49a6-9b47-57754f927de6.png)

I use grass main compiled a couple of days ago:
```
GRASS version: 8.0.dev                                                          
Code revision: 75dd60a69                                                        
Build date: 2021-09-19                                                          
Build platform: x86_64-pc-linux-gnu                                             
GDAL: 3.2.2                                                                     
PROJ: 7.2.1                                                                     
GEOS: 3.9.0                                                                     
SQLite: 3.34.1                                                                  
Python: 3.9.7                                                                   
wxPython: 4.0.7                                                                 
Platform: Linux-5.13.16-200.fc34.x86_64-x86_64-with-glibc2.33
```

**Comment by madi** [comment]
Hi Vero, thanks for checking. I verified and with shp works. 

![Screenshot from 2021-09-23 08-29-42](https://user-images.githubusercontent.com/246068/134463490-21531ab6-fec7-4654-b2c1-dfc1b27a0d78.png)


It doesn't seem to work with Geopackage.

![Screenshot from 2021-09-23 08-27-33](https://user-images.githubusercontent.com/246068/134463507-737e11fe-3a6f-4958-bef1-24dd1339fd67.png)

**Comment by veroandreo** [comment]
It works here also with geopackage... no idea 

![image](https://user-images.githubusercontent.com/20075188/134499801-50063422-9b20-4a92-9a2e-0f68b3e1d6f6.png)

**Comment by tmszi** [comment]
I confirm this issue. According my debug info Import vector data wxGUI dialog internally use `v.external` command for list of available layers including feature type in data source. 

https://github.com/OSGeo/grass/blob/fb4a9754ba0f658dd877123d5071dbe1d233b6d0/gui/wxpython/gui_core/gselect.py#L2239-L2241


If I manually run `v.external` command (`/tmp` dir contains two GeoPackage files) I get error message:

```
GRASS nc_basic_spm_grass7/PERMANENT:~ > v.external -t input=/tmp/
ERROR: Unable to open data source '/tmp/'
```

My GeoPackage files are vector maps exported by GRASS GIS.

`v.external` command internally use [OGR API ](https://gdal.org/api/vector_c_api.html?highlight=ogropen#_CPPv47OGROpenPKciP12OGRSFDriverH) for for list of available layers including feature type in data source.

https://github.com/OSGeo/grass/blob/fb4a9754ba0f658dd877123d5071dbe1d233b6d0/vector/v.external/list.c#L258-L260

It seems these GeoPackages isn't supported by OGR.

**Comment by veroandreo** [comment]
kml is not recognized either... only ESRI shapefiles

**Comment by tmszi** [comment]
> kml is not recognized either... only ESRI shapefiles.

Yes of course, because not all OGR format driver support get list of available layers including feature type in data source if data source is directory (ESRI Shapefile driver supports it).

e.g.

```
GRASS nc_basic_spm_grass7/PERMANENT:~ > v.external -t input=/tmp/elev_pts.gpkg 
Data source </tmp/elev_pts.gpkg> (format 'GPKG') contains 1 layer:
elev_pts,point,1,geom

```
